### DISCUSSION TITLE
[History] allow to navigate to previous interval

### DISCUSSION THREAD

**Discussion by coderbyheart** [hello-nrfcloud/web#918] [Ideas]
I want to see the detail data for the previous hour.

Currently the history chart only allows to switch between different periods ("last hour", "last day", etc):

![image](https://github.com/user-attachments/assets/1fda4914-5790-4683-9d2a-5dcff9dd0f6d)

The component needs to be extended to allow to navigate to previous (and optionally later) periods.

- [ ] add navigation to previous period
- [ ] add navigation to next period (if not on latest)
- [ ] query history API with start time of selected period (depends on https://github.com/hello-nrfcloud/backend/issues/867)

**Comment by ShadowFax1731** [comment]
Can you elaborate?

**Comment by coderbyheart** [comment]
Sure, thanks for taking an interest. I've updated the description above.
